### PR TITLE
add function of get term field's value by field name

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
add function of get term field's value by field name

### CODE REVIEW

File: canal/rows.go [hunk 1]
```diff
@@ -53,6 +53,16 @@ func GetPKValues(table *schema.Table, row []interface{}) ([]interface{}, error)
 	return values, nil
 }
 
+// Get term field's value
+func GetFieldValue(table *schema.Table, field string, row []interface{}) (interface{}, error) {
```
Comment (siddontang) [inline canal/rows.go]: I prefer GetColumnValue to keep the same with other column functions.

**Comment by WangXiangUSTC** [comment]
ok, I have already change field to column

File: canal/rows.go [hunk 2]
```diff
@@ -53,11 +53,11 @@ func GetPKValues(table *schema.Table, row []interface{}) ([]interface{}, error)
...
 
-// Get term field's value
-func GetFieldValue(table *schema.Table, field string, row []interface{}) (interface{}, error) {
-	index := table.FindColumn(field)
+// Get term column's value
+func GetColumnValue(table *schema.Table, column string, row []interface{}) (interface{}, error) {
+	index := table.FindColumn(column)
 	if index == -1 {
-		return nil, errors.Errorf("table %s has no field name %s", table, field)
+		return nil, errors.Errorf("table %s has no field name %s", table, column)
```
Comment (siddontang) [inline canal/rows.go]: no column name

File: canal/rows.go [hunk 3]
```diff
@@ -53,11 +53,11 @@ func GetPKValues(table *schema.Table, row []interface{}) ([]interface{}, error)
 	return values, nil
 }
 
-// Get term field's value
-func GetFieldValue(table *schema.Table, field string, row []interface{}) (interface{}, error) {
-	index := table.FindColumn(field)
+// Get term column's value
```
Comment (siddontang) [inline canal/rows.go]: what does term mean here?

**Pull request merged**